src/opa.js

```diff
@@ -27,7 +27,7 @@ function stringDecoder(mem) {
  */
 function _loadJSON(wasmInstance, memory, value) {
   if (value === undefined) {
-    throw "unable to load undefined value into memory";
+    return 0;
```

Comment:
Interesting. What's the meaning of returning 0 here?

Reply:
0 is basically NULL here, it's going to be `undefined` as far as our Wasm code is concerned.

There's a bit of a backstory here, as in Wasm, 0 is a valid memory address (just the start of it), but the NULL pointer usually is not a valid address... anyways, for memory it's ok; for function pointers, the compiler has to do some extra work IIRC. Doesn't matter much here.

(👉 https://github.com/open-policy-agent/opa/blob/30a06544f26b2dd52df8b785543ccc1ba431b54f/test/wasm/assets/test.js#L72-L76 this is the non-sdk-y JS code we're running in OPA.)